src/RobotMap.h

```diff
@@ -26,6 +26,7 @@ class RobotMap {
 	  static const int kIDIntakeMotor = 5;
 	//..........Arm..........
 	  static const int kIDArmMotor = 10;
+	  static const int kIDArmPot = 3;
```

Comment:
Let's distinguish between CAN IDs and other RoboRio ports. For example, `kIDArmPot` is an analog input on the RoboRio. Maybe use `kAIIDArmPot` or `kAnalogIDArmPot`?